### PR TITLE
Remove licenses from the top of files

### PR DESCRIPTION
**Pull request by wyardley**
I think these aren't needed, since the license is already included in the project, thoughts?

### CODE REVIEW

File: lib/puppet/provider/grafana.rb [hunk 1]
```diff
@@ -1,17 +1,3 @@
-#    Copyright 2015 Mirantis, Inc.
```
Comment (bastelfreak) [inline lib/puppet/provider/grafana.rb]: I am not sure if we can remove the copyright. Any apache-2 license experts around?

Comment (wyardley) [reply lib/puppet/provider/grafana.rb]: I could also put the copyright back in, but let's see if anyone knows.

**Comment by bastelfreak** [comment]
+1 for merging from my side. @wyardley do you want more feedback or merge it?

**Comment by wyardley** [comment]
@bastelfreak IANAL, but merging it works for me.

**Pull request merged**